overrides/scripts/Alchemy.zs

```diff
@@ -19,10 +19,6 @@ recipes.addShaped(<thermalfoundation:tool.shears_wood>, [[null,<minecraft:stick>
 //Rubber Tree
 recipes.addShapeless(<gregtech:sapling>, [<ore:treeSapling>,<gregtech:meta_item_1:32627>]);
 
-//Fertilizer 
-recipes.remove(<forestry:fertilizer_compound>);
-recipes.addShaped(<forestry:fertilizer_compound> * 8, [[<minecraft:sand>,<gregtech:meta_item_1:8226>,<minecraft:sand>]]);

```

Comment:
Could you please change this recipe addition to produce the AA fertilizer, `<actuallyadditions:item_fertilizer>` and give the recipe a name.